ThunderCloud.xcodeproj/project.pbxproj

```diff
@@ -1849,6 +1849,7 @@
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = arm64;
```

Comment:
If low LOW, we should get this into some `xcconfig` or something so it can be easily shared and reused for different projects. This is how you ran it on the simulator?

Reply:
This is how I got the project to compile using `Carthage` 👀

Reply:
Hmmm, I was able to get it compile, hence the `v3.0.0` binary created